src/wiki/core/wiki_paginate.py

```diff
@@ -0,0 +1,30 @@
+from django.core.paginator import Paginator
+
+class WikiPaginator(Paginator):
+
+    def __init__(self, *args, **kwargs):
+        """
+        :param side_pages: How many pages should be shown before and after the current page
+        """
+        self.side_pages = kwargs.pop('side_pages', 4)
+        super(WikiPaginator, self).__init__(*args, **kwargs)
+
+    def _get_page(self, *args, **kwargs):
+        self.curPage = super(WikiPaginator, self)._get_page(*args, **kwargs)
```

Comment:
I tried reordering this a bit 

1. We don't overwrite an weak internal method
1. renamed `curPage` to something with underscores, camelCase isn't used in Python normally :)

It's in https://github.com/floemker/django-wiki/pull/4